### DISCUSSION TITLE
Is this provider still maintained?

### DISCUSSION THREAD

**Discussion by JanKoppe** [kreuzwerker/terraform-provider-docker#577] [General]
I've noticed that @mavogel has apparently left kreuzwerker GmbH, and did most of the project maintenance. Since then, issues and PRs seem to not be handled by anyone else from kreuzwerker.

It would be nice to know, if there is some new leadership on this project planned, and when we can maybe count on PRs and issues being handled again. I do understand that this is an open-source project and you have zero obligations to accept/reject PRs, or help out with issues. It is a bit difficult not knowing at all what the future of this provider is going to be, though.

A few words about what's going on and what (maybe) is planned for the future would be very appreciated!

And: thanks for all the work so far :)

**Comment by tinexw** [comment]
Hi @JanKoppe,

First of all, sorry for the late reply.

We are unfortunately still in the process of deciding how to handle this project. As of now, we unfortunately do not have a new core maintainer lined up. If there is a specific PR or issue you would like to be handled, please let us know and we will try to handle it short notice. Again, really sorry for the slow speed here.

Best,

Tine

**Reply by mkgeneral** [reply]
Hi @tinexw ,

Is it possible to look at [issue 557](https://github.com/kreuzwerker/terraform-provider-docker/discussions/557) when an image is either uploaded to a private registry or an image is read from it?

This issue prevents from using docker provider. 

Thank you!

**Reply by tinexw** [reply]
Hi @mkgeneral,

sorry again for the late reply. We unfortunately cannot look into the issue ourselves in the foreseeable future. However, if you provide a pull request we might be able to review and merge it in.

**Comment by MiguelNdeCarvalho** [comment]
Hey, any update on this?

**Comment by Dr-Shadow** [comment]
Hello @tinexw, I would like to have an update about the situation.

**Comment by IlyesDemineExtVeolia** [comment]
Hello, Any news ? @tinexw

**Comment by Junkern** [comment]
I am a former maintainer and as mentioned [here](https://github.com/kreuzwerker/terraform-provider-docker/issues/670#issuecomment-2790937291) I managed to get push permissions. Let's get this thing started again

**Reply by Lyessi78** [reply]
@Junkern Very good news. It was really frustrating knowing that this provider is used by the official AWS lambda module and was not updated. 

Do you have any ideas on which topics will be covered first? 

This bug is really annoying on my side https://github.com/kreuzwerker/terraform-provider-docker/issues/603

Good luck with resuming repo 🙂

**Reply by Junkern** [reply]
I will go step by step and fix the most obvious/easiest bugs first. I am doing this in my free time, so I sadly can't give promises on timelines.

**Comment by Junkern** [comment]
Provider is still maintained. Closing this discussions
